app/jobs/integration_customers/create_job.rb

```diff
@@ -5,6 +5,7 @@ class CreateJob < ApplicationJob
     queue_as 'integrations'
 
     retry_on LagoHttpClient::HttpError, wait: :polynomially_longer, attempts: 3
+    retry_on BaseService::ThrottlingError, wait: :polynomially_longer, attempts: 25
```

Comment:
I feel like this is generic enough that we could consider moving it to `ApplicationJob` or even the `ConcurrencyThrottlable` module

Reply:
Now it is in the `BaseService` where all the other erros are defined as well. 
We can define it it the `throttling.rb` initializer, since it is just:

```
class ThrottlingError < StandardError; end
```

I wouldn't put it in `ApplicationJob ` because it can be raised even outside of a job. It is dependent on redis not sidekiq (or active job)